backend/tests/regression/answer_quality/launch_search_test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
mmm not a big fan of this type of approach, we can just have a set of exports to use and it can just be a command we share internally. If someone else wants to run this eval, they can set up their own model server. But also nobody else can really do this since they would need a dataset from Relari and also do all the cleanup and etc... at that point I'm sure they would have reached out to us